DOC: Update Security document to clarify "bug bounty" scope (#7223)
Slicer/Slicer: SECURITY.md

> [!IMPORTANT]
> While we may not be able to offer legally binding commitments, we will do our best in addressing any reported security concerns.

> [!WARNING]

> > While we greatly appreciate contributions to make Slicer more secure, please be aware that Slicer is **NOT** part of an official bug bounty program, and no financial compensation is offered for such contributions.

Reviewer: What do you mean by "official"? Maybe instead we could write something like this:

```suggestion
> While we greatly appreciate contributions to make Slicer more secure, please be aware that the Slicer community does not run a bug bounty program, and no financial compensation is offered for such contributions.
```

Author: > What do you mean by "official"?

I adopted the term "official" from https://github.com/ossf/oss-vulnerability-guide/blob/main/maintainer-guide.md#what-does-the-vulnerability-reporter-want

The proposed change make sense.
